lib/skate_web/controllers/route_controller.ex

```diff
@@ -0,0 +1,8 @@
+defmodule SkateWeb.RouteController do
+  use SkateWeb, :controller
+
+  def index(conn, _params) do
+    routes = Gtfs.Query.all_routes()
+    json(conn, %{data: routes})
```

Comment:
Could `routes` be a more descriptive key than `data`?

Reply:
You'd know it's routes because it's from the `/routes` endpoint. This is just to get around the restriction that you can only return objects as the root of a json endpoint. Since all the endpoints will need this, I think it will be easier to always use `data:` rather than try to remember which endpoint uses which field name. This is consistent with the way the v3 api uses the `data` field in the json api format.

Reply:
Ok, that makes sense.